xml/cap_depl_azure.xml

```diff
@@ -634,7 +634,167 @@ kube:
+        
+&prompt.user;helm repo update
+
+&prompt.user;helm install svc-cat/catalog --name catalog --namespace catalog --set rbacEnable=false --set controllerManager.healthcheck.enabled=false --set apiserver.storage.etcd.persistence.storageClass=default
+
+&prompt.user;kubectl get apiservice
+
+&prompt.user;helm repo add azure https://kubernetescharts.blob.core.windows.net/azure
+
+&prompt.user;helm repo update
+</screen>
+
+<para>
+Set up the service broker with your variables:
+</para>
+<screen>
+&prompt.user;helm install azure/open-service-broker-azure \
+--name <replaceable>osba</replaceable> \
+--namespace <replaceable>osba</replaceable> \
+--set azure.subscriptionId=${AZURE_SUBSCRIPTION_ID} \
+--set azure.tenantId=${AZURE_TENANT_ID} \
+--set azure.clientId=${AZURE_CLIENT_ID} \
+--set azure.clientSecret=${AZURE_CLIENT_SECRET} \
+--set azure.defaultLocation=${LOCATION} \
+--set redis.persistence.storageClass=default \
+--set basicAuth.username=$(tr -dc 'a-zA-Z0-9' &lt; /dev/urandom | head -c 16) \
+--set basicAuth.password=$(tr -dc 'a-zA-Z0-9' &lt; /dev/urandom | head -c 16)
+
+&prompt.user;cf create-service-broker azure $(kubectl get deployment osba-open-service-broker-azure \
+--namespace osba -o jsonpath='{.spec.template.spec.containers[0].env[?(@.name == "BASIC_AUTH_USERNAME")].value}') $(kubectl get secret --namespace osba osba-open-service-broker-azure -o jsonpath='{.data.basic-auth-password}' | base64 -d) http://osba-open-service-broker-azure.osba
+</screen>
+<para>
+    Modify the database:
+</para>
+<screen>
+&prompt.user;kubectl exec -it mysql-0 --namespace scf -- env TERM=xterm-256color COLUMNS=135 LINES=62 /bin/bash
+  
+# mysql --defaults-file=/var/vcap/jobs/mysql/config/mylogin.cnf ccdb
+    MariaDB [ccdb]> ALTER TABLE ccdb.service_plans MODIFY COLUMN description VARCHAR(1024) COLLATE utf8_bin ;
+    MariaDB [ccdb]> exit
+
+# exit
+</screen>
+<para>
+    Register the service broker in &scf;:
+</para>
+<screen>
+&prompt.user;cf login
+
+&prompt.user;cf create-service-broker azure \
+$(kubectl get deployment osba-open-service-broker-azure \
+-namespace osba -o jsonpath='{.spec.template.spec.containers[0].env[?(@.name == "BASIC_AUTH_USERNAME")].value}') \
+$(kubectl get secret --namespace osba osba-open-service-broker-azure -o jsonpath='{.data.basic-auth-password}' | \
+base64 -d) http://osba-open-service-broker-azure.osba
+
+&prompt.user;cf service-access -b azure | \
+awk '($2 ~ /basic/) { system("cf enable-service-access " $1 " -p " $2)}'
+</screen>
+<para>
+    Test your new service broker with an example Rails app. First create a space
+    and org for your test app:
+</para>
+<screen>
+&prompt.user;cf create-org <replaceable>testorg</replaceable>
+
+&prompt.user;cf create-space <replaceable>scftest</replaceable> -o <replaceable>testorg</replaceable>
+
+yh8lgt_987654/ gprompt.user;cf target -o "<replaceable>testorg</replaceable>" -s "<replaceable>scftest</replaceable>"
```

Comment:
Not sure what happened at the start there :D

Reply:
The cat. Always it is the cat's fault.